Course_Part_03/src/Assingment08c.java

```diff
@@ -2,6 +2,7 @@
     public static void main(String[] args) {
         int a = 10;
         int b = -21;
+
         System.out.println ("Het kleinste getal is " + ((a < b) ? a : b));
         System.out.println ("Het grootste getal is " + ((a > b) ? a : b));
         System.out.println ("De absolute waarde van b is " + ((b > 0) ? b : Math.abs (b)));
```

Comment:
Math.abs gebruiken is helemaal prima, maar hoe zou je het op kunnen lossen zonder er gebruik van te maken?